## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 507331`
Commit: `1df71e28241d47c99a531dda6ef70ad9e32ac15d`

---
### `aarch64-darwin`
<details>
  <summary>:x: 10 packages failed to build:</summary>
  <ul>
    <li>python313Packages.mmcv</li>
    <li>python313Packages.mmcv.dist</li>
    <li>python313Packages.mmengine</li>
    <li>python313Packages.mmengine.dist</li>
    <li>python313Packages.torchtune</li>
    <li>python313Packages.torchtune.dist</li>
    <li>python314Packages.mmcv</li>
    <li>python314Packages.mmcv.dist</li>
    <li>python314Packages.mmengine</li>
    <li>python314Packages.mmengine.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 150 packages built:</summary>
  <ul>
    <li>langgraph-cli (python313Packages.langgraph-cli)</li>
    <li>langgraph-cli.dist (python313Packages.langgraph-cli.dist)</li>
    <li>mlflow-server</li>
    <li>mlflow-server.dist</li>
    <li>python313Packages.databricks-sdk</li>
    <li>python313Packages.databricks-sdk.dist</li>
    <li>python313Packages.great-tables</li>
    <li>python313Packages.great-tables.dist</li>
    <li>python313Packages.langchain</li>
    <li>python313Packages.langchain-anthropic</li>
    <li>python313Packages.langchain-anthropic.dist</li>
    <li>python313Packages.langchain-aws</li>
    <li>python313Packages.langchain-aws.dist</li>
    <li>python313Packages.langchain-azure-dynamic-sessions</li>
    <li>python313Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python313Packages.langchain-chroma</li>
    <li>python313Packages.langchain-chroma.dist</li>
    <li>python313Packages.langchain-classic</li>
    <li>python313Packages.langchain-classic.dist</li>
    <li>python313Packages.langchain-community</li>
    <li>python313Packages.langchain-community.dist</li>
    <li>python313Packages.langchain-core</li>
    <li>python313Packages.langchain-core.dist</li>
    <li>python313Packages.langchain-deepseek</li>
    <li>python313Packages.langchain-deepseek.dist</li>
    <li>python313Packages.langchain-experimental</li>
    <li>python313Packages.langchain-experimental.dist</li>
    <li>python313Packages.langchain-fireworks</li>
    <li>python313Packages.langchain-fireworks.dist</li>
    <li>python313Packages.langchain-google-genai</li>
    <li>python313Packages.langchain-google-genai.dist</li>
    <li>python313Packages.langchain-groq</li>
    <li>python313Packages.langchain-groq.dist</li>
    <li>python313Packages.langchain-huggingface</li>
    <li>python313Packages.langchain-huggingface.dist</li>
    <li>python313Packages.langchain-mistralai</li>
    <li>python313Packages.langchain-mistralai.dist</li>
    <li>python313Packages.langchain-mongodb</li>
    <li>python313Packages.langchain-mongodb.dist</li>
    <li>python313Packages.langchain-ollama</li>
    <li>python313Packages.langchain-ollama.dist</li>
    <li>python313Packages.langchain-openai</li>
    <li>python313Packages.langchain-openai.dist</li>
    <li>python313Packages.langchain-perplexity</li>
    <li>python313Packages.langchain-perplexity.dist</li>
    <li>python313Packages.langchain-tests</li>
    <li>python313Packages.langchain-tests.dist</li>
    <li>python313Packages.langchain-text-splitters</li>
    <li>python313Packages.langchain-text-splitters.dist</li>
    <li>python313Packages.langchain-xai</li>
    <li>python313Packages.langchain-xai.dist</li>
    <li>python313Packages.langchain.dist</li>
    <li>python313Packages.langgraph</li>
    <li>python313Packages.langgraph-checkpoint</li>
    <li>python313Packages.langgraph-checkpoint-mongodb</li>
    <li>python313Packages.langgraph-checkpoint-mongodb.dist</li>
    <li>python313Packages.langgraph-checkpoint-postgres</li>
    <li>python313Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python313Packages.langgraph-checkpoint-sqlite</li>
    <li>python313Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python313Packages.langgraph-checkpoint.dist</li>
    <li>python313Packages.langgraph-prebuilt</li>
    <li>python313Packages.langgraph-prebuilt.dist</li>
    <li>python313Packages.langgraph-runtime-inmem</li>
    <li>python313Packages.langgraph-runtime-inmem.dist</li>
    <li>python313Packages.langgraph-store-mongodb</li>
    <li>python313Packages.langgraph-store-mongodb.dist</li>
    <li>python313Packages.langgraph.dist</li>
    <li>python313Packages.langsmith</li>
    <li>python313Packages.langsmith.dist</li>
    <li>python313Packages.mlflow</li>
    <li>python313Packages.mlflow.dist</li>
    <li>python313Packages.sagemaker-mlflow</li>
    <li>python313Packages.sagemaker-mlflow.dist</li>
    <li>python313Packages.shiny</li>
    <li>python313Packages.shiny.dist</li>
    <li>python314Packages.databricks-sdk</li>
    <li>python314Packages.databricks-sdk.dist</li>
    <li>python314Packages.great-tables</li>
    <li>python314Packages.great-tables.dist</li>
    <li>python314Packages.langchain</li>
    <li>python314Packages.langchain-anthropic</li>
    <li>python314Packages.langchain-anthropic.dist</li>
    <li>python314Packages.langchain-aws</li>
    <li>python314Packages.langchain-aws.dist</li>
    <li>python314Packages.langchain-azure-dynamic-sessions</li>
    <li>python314Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python314Packages.langchain-chroma</li>
    <li>python314Packages.langchain-chroma.dist</li>
    <li>python314Packages.langchain-classic</li>
    <li>python314Packages.langchain-classic.dist</li>
    <li>python314Packages.langchain-community</li>
    <li>python314Packages.langchain-community.dist</li>
    <li>python314Packages.langchain-core</li>
    <li>python314Packages.langchain-core.dist</li>
    <li>python314Packages.langchain-deepseek</li>
    <li>python314Packages.langchain-deepseek.dist</li>
    <li>python314Packages.langchain-experimental</li>
    <li>python314Packages.langchain-experimental.dist</li>
    <li>python314Packages.langchain-fireworks</li>
    <li>python314Packages.langchain-fireworks.dist</li>
    <li>python314Packages.langchain-google-genai</li>
    <li>python314Packages.langchain-google-genai.dist</li>
    <li>python314Packages.langchain-groq</li>
    <li>python314Packages.langchain-groq.dist</li>
    <li>python314Packages.langchain-huggingface</li>
    <li>python314Packages.langchain-huggingface.dist</li>
    <li>python314Packages.langchain-mistralai</li>
    <li>python314Packages.langchain-mistralai.dist</li>
    <li>python314Packages.langchain-mongodb</li>
    <li>python314Packages.langchain-mongodb.dist</li>
    <li>python314Packages.langchain-ollama</li>
    <li>python314Packages.langchain-ollama.dist</li>
    <li>python314Packages.langchain-openai</li>
    <li>python314Packages.langchain-openai.dist</li>
    <li>python314Packages.langchain-perplexity</li>
    <li>python314Packages.langchain-perplexity.dist</li>
    <li>python314Packages.langchain-tests</li>
    <li>python314Packages.langchain-tests.dist</li>
    <li>python314Packages.langchain-text-splitters</li>
    <li>python314Packages.langchain-text-splitters.dist</li>
    <li>python314Packages.langchain-xai</li>
    <li>python314Packages.langchain-xai.dist</li>
    <li>python314Packages.langchain.dist</li>
    <li>python314Packages.langgraph</li>
    <li>python314Packages.langgraph-checkpoint</li>
    <li>python314Packages.langgraph-checkpoint-mongodb</li>
    <li>python314Packages.langgraph-checkpoint-mongodb.dist</li>
    <li>python314Packages.langgraph-checkpoint-postgres</li>
    <li>python314Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python314Packages.langgraph-checkpoint-sqlite</li>
    <li>python314Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python314Packages.langgraph-checkpoint.dist</li>
    <li>python314Packages.langgraph-cli</li>
    <li>python314Packages.langgraph-cli.dist</li>
    <li>python314Packages.langgraph-prebuilt</li>
    <li>python314Packages.langgraph-prebuilt.dist</li>
    <li>python314Packages.langgraph-runtime-inmem</li>
    <li>python314Packages.langgraph-runtime-inmem.dist</li>
    <li>python314Packages.langgraph-store-mongodb</li>
    <li>python314Packages.langgraph-store-mongodb.dist</li>
    <li>python314Packages.langgraph.dist</li>
    <li>python314Packages.langsmith</li>
    <li>python314Packages.langsmith.dist</li>
    <li>python314Packages.mlflow</li>
    <li>python314Packages.mlflow.dist</li>
    <li>python314Packages.sagemaker-mlflow</li>
    <li>python314Packages.sagemaker-mlflow.dist</li>
    <li>python314Packages.shiny</li>
    <li>python314Packages.shiny.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>python313Packages.mmengine</summary>
<pre>  /nix/var/nix/b/1qlk034lxy5fhncl9lbf5pnwgv/source/mmengine/visualization/visualizer.py:831: UserWarning: Warning: The polygon is out of bounds, the drawn polygon may not be in the image
    warnings.warn(

tests/test_visualizer/test_visualizer.py::TestVisualizer::test_init
  /nix/var/nix/b/1qlk034lxy5fhncl9lbf5pnwgv/source/mmengine/utils/manager.py:113: UserWarning: &lt;class &#x27;mmengine.visualization.visualizer.Visualizer&#x27;&gt; instance named of test_save_dir has been created, the method `get_instance` should not accept any other arguments
    warnings.warn(

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/test_analysis/test_jit_analysis.py::TestJitModelAnalysis::test_disable_warnings - ModuleNotFoundError: No module named &#x27;torchvision&#x27;
FAILED tests/test_hooks/test_early_stopping_hook.py::TestEarlyStoppingHook::test_after_val_epoch - ModuleNotFoundError: No module named &#x27;torchvision&#x27;
FAILED tests/test_optim/test_scheduler/test_lr_scheduler.py::TestLRScheduler::test_overwrite_optimzer_step - ModuleNotFoundError: No module named &#x27;torchvision&#x27;
FAILED tests/test_optim/test_scheduler/test_lr_scheduler.py::TestLRScheduler::test_scheduler_before_optim_warning - ModuleNotFoundError: No module named &#x27;torchvision&#x27;
FAILED tests/test_optim/test_scheduler/test_momentum_scheduler.py::TestMomentumScheduler::test_overwrite_optimzer_step - ModuleNotFoundError: No module named &#x27;torchvision&#x27;
FAILED tests/test_optim/test_scheduler/test_momentum_scheduler.py::TestMomentumScheduler::test_scheduler_before_optim_warning - ModuleNotFoundError: No module named &#x27;torchvision&#x27;
FAILED tests/test_optim/test_scheduler/test_param_scheduler.py::TestParameterScheduler::test_overwrite_optimzer_step - ModuleNotFoundError: No module named &#x27;torchvision&#x27;
FAILED tests/test_optim/test_scheduler/test_param_scheduler.py::TestParameterScheduler::test_scheduler_before_optim_warning - ModuleNotFoundError: No module named &#x27;torchvision&#x27;
FAILED tests/test_optim/test_scheduler/test_param_scheduler.py::TestParameterSchedulerOptimWrapper::test_overwrite_optimzer_step - ModuleNotFoundError: No module named &#x27;torchvision&#x27;
FAILED tests/test_optim/test_scheduler/test_param_scheduler.py::TestParameterSchedulerOptimWrapper::test_scheduler_before_optim_warning - ModuleNotFoundError: No module named &#x27;torchvision&#x27;
= 10 failed, 710 passed, 73 skipped, 137 deselected, 430 warnings in 82.40s (0:01:22) =</pre>
</details>
<details>
<summary>python314Packages.mmengine</summary>
<pre>  /nix/var/nix/b/5dn28rq34vpbw93xaqfc2wnq4q/source/mmengine/visualization/visualizer.py:831: UserWarning: Warning: The polygon is out of bounds, the drawn polygon may not be in the image
    warnings.warn(

tests/test_visualizer/test_visualizer.py::TestVisualizer::test_init
  /nix/var/nix/b/5dn28rq34vpbw93xaqfc2wnq4q/source/mmengine/utils/manager.py:113: UserWarning: &lt;class &#x27;mmengine.visualization.visualizer.Visualizer&#x27;&gt; instance named of test_save_dir has been created, the method `get_instance` should not accept any other arguments
    warnings.warn(

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/test_analysis/test_jit_analysis.py::TestJitModelAnalysis::test_disable_warnings - ModuleNotFoundError: No module named &#x27;torchvision&#x27;
FAILED tests/test_hooks/test_early_stopping_hook.py::TestEarlyStoppingHook::test_after_val_epoch - ModuleNotFoundError: No module named &#x27;torchvision&#x27;
FAILED tests/test_optim/test_scheduler/test_lr_scheduler.py::TestLRScheduler::test_overwrite_optimzer_step - ModuleNotFoundError: No module named &#x27;torchvision&#x27;
FAILED tests/test_optim/test_scheduler/test_lr_scheduler.py::TestLRScheduler::test_scheduler_before_optim_warning - ModuleNotFoundError: No module named &#x27;torchvision&#x27;
FAILED tests/test_optim/test_scheduler/test_momentum_scheduler.py::TestMomentumScheduler::test_overwrite_optimzer_step - ModuleNotFoundError: No module named &#x27;torchvision&#x27;
FAILED tests/test_optim/test_scheduler/test_momentum_scheduler.py::TestMomentumScheduler::test_scheduler_before_optim_warning - ModuleNotFoundError: No module named &#x27;torchvision&#x27;
FAILED tests/test_optim/test_scheduler/test_param_scheduler.py::TestParameterScheduler::test_overwrite_optimzer_step - ModuleNotFoundError: No module named &#x27;torchvision&#x27;
FAILED tests/test_optim/test_scheduler/test_param_scheduler.py::TestParameterScheduler::test_scheduler_before_optim_warning - ModuleNotFoundError: No module named &#x27;torchvision&#x27;
FAILED tests/test_optim/test_scheduler/test_param_scheduler.py::TestParameterSchedulerOptimWrapper::test_overwrite_optimzer_step - ModuleNotFoundError: No module named &#x27;torchvision&#x27;
FAILED tests/test_optim/test_scheduler/test_param_scheduler.py::TestParameterSchedulerOptimWrapper::test_scheduler_before_optim_warning - ModuleNotFoundError: No module named &#x27;torchvision&#x27;
= 10 failed, 710 passed, 73 skipped, 137 deselected, 499 warnings in 79.94s (0:01:19) =</pre>
</details>
</details>

--- a/pkgs/development/python-modules/langchain-core/default.nix
+++ b/pkgs/development/python-modules/langchain-core/default.nix
@@ -36,14 +36,14 @@
 
 buildPythonPackage (finalAttrs: {
   pname = "langchain-core";
-  version = "1.2.22";
+  version = "1.2.27";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "langchain-ai";
     repo = "langchain";
     tag = "langchain-core==${finalAttrs.version}";
-    hash = "sha256-jaoZtRNHAXbt/a9hx4zPGEYppT/Kxu0MPn/cOj5kLP4=";
+    hash = "sha256-WdC43SUVgteNQLNMD+yEkD8TyyCFGXkYTi5moZMEmVo=";
   };
 
   sourceRoot = "${finalAttrs.src.name}/libs/core";

--- a/pkgs/development/python-modules/langgraph-prebuilt/default.nix
+++ b/pkgs/development/python-modules/langgraph-prebuilt/default.nix
@@ -14,6 +14,7 @@
   # tests
   langgraph-checkpoint-postgres,
   langgraph-checkpoint-sqlite,
+  langgraph-sdk,
   postgresql,
   postgresqlTestHook,
   psycopg,
@@ -31,14 +32,14 @@
 # It exists so the langgraph team can iterate on it without having to rebuild langgraph.
 buildPythonPackage (finalAttrs: {
   pname = "langgraph-prebuilt";
-  version = "1.0.8";
+  version = "1.0.9";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "langchain-ai";
     repo = "langgraph";
     tag = "prebuilt==${finalAttrs.version}";
-    hash = "sha256-Gsh2bCcity0zf9A+FENxwktK5j3WhQOG/jZmhJ18KVE=";
+    hash = "sha256-B9ntvj6qoKd63W/cPkevtfHbv8XUxmkaWupxURA8C68=";
   };
 
   sourceRoot = "${finalAttrs.src.name}/libs/prebuilt";
@@ -48,6 +49,7 @@ buildPythonPackage (finalAttrs: {
   dependencies = [
     langchain-core
     langgraph-checkpoint
+    langgraph-sdk
   ];
 
   skipPythonImportsCheck = true; # This will be packaged with langgraph

--- a/pkgs/development/python-modules/langgraph-runtime-inmem/default.nix
+++ b/pkgs/development/python-modules/langgraph-runtime-inmem/default.nix
@@ -14,14 +14,14 @@
 
 buildPythonPackage (finalAttrs: {
   pname = "langgraph-runtime-inmem";
-  version = "0.26.0";
+  version = "0.27.3";
   pyproject = true;
 
   # Not available in any repository
   src = fetchPypi {
     pname = "langgraph_runtime_inmem";
     inherit (finalAttrs) version;
-    hash = "sha256-ucWH0TOTIKKlSlcKIa7K9Z7rxL4HzvHYpbA18/LGHWo=";
+    hash = "sha256-jylX7GXEJFJr5pBxCnD2W/kG3dkqS9IFS7VhyPlErJE=";
   };
 
   build-system = [ hatchling ];

